Frontend-Development/README.md

```diff
@@ -57,6 +57,7 @@
 - [Learn JavaScript - Codecademy](https://www.codecademy.com/learn/introduction-to-javascript)
 - [Free JS Courses for Beginners - FreeCodeCamp](https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/)
 - [Free JS Practice website to master JS - exercism](https://exercism.org/)
+- [JS Hands-on project based interactive course - Scrimba](https://scrimba.com/)
```

Comment:
Changing the link from `https://scrimba.com/` to `https://scrimba.com/allcourses?topic=javascript` would be great in my opinion!